index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Js task</title>
+</head>
+<body>
+  <script src="./index.js"></script>
+</body>
+</html>
```

Comment:
eof (end of file always should be an empty string) that's why github shows the error